# This is a BitKeeper generated patch for the following project:
# Project Name: Linux kernel tree
# This patch format is intended for GNU patch command version 2.5 or higher.
# This patch includes the following deltas:
#	           ChangeSet	1.1367  -> 1.1368
#	include/linux/swap.h	1.79    -> 1.80
#
# The following is the BitKeeper ChangeSet Log
# --------------------------------------------
# 03/10/16	noah@caltech.edu	1.1368
# The linux/swap.h header uses the page_cache_release and release_pages
# functions declared in linux/pagemap.h when CONFIG_SWAP is disabled.  Add
# an include of linux/pagemap.h so swap.h finds those declarations on
# architectures that don't include pagemap.h indirectly.
# --------------------------------------------
#
diff -Nru a/include/linux/swap.h b/include/linux/swap.h
--- a/include/linux/swap.h	Fri Oct 17 13:40:06 2003
+++ b/include/linux/swap.h	Fri Oct 17 13:40:06 2003
@@ -4,6 +4,7 @@
 #include <linux/config.h>
 #include <linux/spinlock.h>
 #include <linux/linkage.h>
+#include <linux/pagemap.h>
 #include <linux/mmzone.h>
 #include <linux/list.h>
 #include <linux/sched.h>
